### ISSUE TITLE
Real machine commissioning: bundling failed: Error: Unable to resolve module `AccessibilityInfo`

### ISSUE THREAD

**Issue by kingangelAOA** [facebook/react-native#17686]
### Is this a bug report?

(write your answer here)

### Have you read the [Contributing Guidelines](https://facebook.github.io/react-native/docs/contributing.html)?

(Write your answer here.)

### Environment

when：
jsCodeLocation = [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index" fallbackResource:nil];

to

jsCodeLocation = [NSURL URLWithString:@"http://192.168.0.101:8081/index.bundle"];

error:
error: bundling failed: Error: Unable to resolve module `AccessibilityInfo` from `/Users/kingangelTOT/Application/git_work/node_work/universal/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: Module does not exist in the module map

### Steps to Reproduce

(Write your steps here:)

1.
2.
3.

### Expected Behavior

(Write what you thought would happen.)

### Actual Behavior

(Write what happened. Add screenshots!)

### Reproducible Demo

(Paste the link to an example project and exact instructions to reproduce the issue.)

**Comment by lwinkyawmyat** [comment]
@facebook-github-bot no-template

**Comment by kingangelAOA** [comment]
The report was revised.

**Comment by aMarCruz** [comment]
I have the same issue in Android.

RN: 0.55.2
buildToolsVersion 27.0.3
compileSDKVersion 26
Linux Mint 18.3
